README.md

```diff
@@ -0,0 +1,2 @@
+# neocomplexx
+Create a server with nodejs and express
```

Comment:
Please rewrite this file following the minimum standar: `Install` , `Usage` ,`Contribuiting` y `Licence`  
take a look at the following Lik https://github.com/RichardLitt/standard-readme/blob/master/example-readmes/minimal-readme.md
:three:  para levantar el servidor, lo vamos a hacer con el siguiente comando: `npm run start` .  Esto significa que vas a tener que introducir un comando en `package.json`en la seccion `scripts`